src/types-and-traits.rst

```diff
@@ -177,7 +177,7 @@ Boolean :t:`value` ``false`` has bit pattern ``0x00``. Boolean :t:`value`
 ``true`` has bit pattern ``0x01``.
 
 :dp:`fls_7nd5tixyqir8`
-The following operations are defined on :t:`type` :c:`bool`:
```

Comment:
I think the former phrasing was better, operations are defined on the types (as the types are implementing the correspodning traits)